src/components/UserInfo/UserInfo.tsx

```diff
@@ -1 +1,17 @@
-export const UserInfo = () => {};
+import { User } from '../../types/User';
+
+type Props = {
+  user: User | null;
+};
+
+export const UserInfo: React.FC<Props> = ({ user }) => {
+  if (user === null) {
+    return;
```

Comment:
When `user` is `null`, the component should return `null` instead of returning nothing. This ensures that the component always returns a valid React element or `null`.